Jellyfin.Server/Program.cs

```diff
@@ -76,6 +76,9 @@ private static async Task StartApp(StartupOptions options)
             // $JELLYFIN_LOG_DIR needs to be set for the logger configuration manager
             Environment.SetEnvironmentVariable("JELLYFIN_LOG_DIR", appPaths.LogDirectoryPath);
 
+            // As far as Jellyfin is concerned the Environment path should also include wherever Jellyfin is running from
+            Environment.SetEnvironmentVariable("PATH", Environment.GetEnvironmentVariable("PATH") + Path.PathSeparator + Environment.CurrentDirectory);
```

Comment:
Wouldn't it be better with Path.Combine?

Reply:
Path.Combine makes directory paths. What I need to do is append the current directory after a semicolon (on windows at least, using Path.PathSeperator because it can vary) to the %PATH% environment variable. Path.Combine won't do me any good here.